README.md

```diff
@@ -1,27 +1,9 @@
 # postfix
```

Comment:
`README.md` is suppose to be rendered: so `mv` this as a template and provide `README.md` rendered

Reply:
No, It is a bug in PR. I don't want to have README.md rendered. Will fix it.

Reply:
No, we need to have README.md rendered because then github will not display anything:

https://github.com/container-images/container-image-template#generated-files

Reply:
You are right.